src/components/ebay-chips-combobox/index.marko

```diff
@@ -43,6 +44,7 @@ $ const options = [...option || []].map((o) => o.text)
         onExpand("emit", "expand")
         onCollapse("emit", "collapse")
         disabled=disabled
+        dropdown-element=() => component.getEl("root")
```

Comment:
I think it should be okay in this case, but passing a function like this is a sketchy pattern in Marko 5 because I believe it will cause problems if the function is called during SSR. Not sure if I can think of anything better, but it is something to note.

Reply:
I did speak with Dylan about this approach. So it seems the only way